sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.3.1 (Unreleased)
 
+- Updated Azure Storage Service API version to 2020-04-08.
+- Added a new interface `BlockBlobClient.syncUploadFromURL()` to support creating a new Block Blob where the contents of the blob are read from a given URL.
```

Comment:
Would better explain to user what's diff between syncCopyFromUrl and syncUploadFromURL.

Reply:
This is just a change log. Users shouldn't depend on it for documentation.